File: class22/java/AP2201-Subrotinas.java
public class Sub {
    static void exibirMenu() {
        System.out.println("1 - Sacar\n2 - Depositar\n3 - Sair");
    }
    static int dobro(int n) { return n * 2; }
    public static void main(String[] args) {
        exibirMenu();
        System.out.println("Dobro de 5 = " + dobro(5));
    }
}

File: class22/cpp/AP2201-Subrotinas.cpp
#include <iostream>
void exibirMenu() {
    std::cout << "1 - Sacar\n2 - Depositar\n3 - Sair" << std::endl;
}
int dobro(int n) { return n * 2; }
int main() {
    exibirMenu();
    std::cout << "Dobro de 5 = " << dobro(5) << std::endl;
    return 0;
}
